index.html

```diff
@@ -15,14 +15,69 @@
     <link rel="stylesheet" href="css/layout.css" />
     <link rel="stylesheet" href="css/styles.css" />
   </head>
+
   <body>
-    <header>
-      <div class="container"></div>
+    <header class="purple-background">
+      <div class="container column-full">
+        <div class="code-journal-header">Code Journal</div>
+      </div>
     </header>
+
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div class="body-main-header row">New Entry</div>
+
+      <form class="form">
+        <div data-view="entry-form column-full">
```

Comment:
Your `div` on line 27 and your `form` on line 29 should be inside the `div` on line 30. You want the `div` with the `data-view="entry-form"` to be the parent to all of this content